tests/util/tracer/blockscout_tracer.min.json

```diff
@@ -1,3 +1,3 @@
error\n        } else {\n            result.createdContractAddressHash = toHex(ctx.to);\n            result.createdContractCode = toHex(db.getCode(ctx.to));\n        }\n    },\n\n    error(ctx) {\n        var error;\n\n        const bottomCall = this.bottomCall();\n        const bottomCallError = bottomCall.error;\n\n        if (bottomCallError !== undefined) {\n            error = bottomCallError;\n        } else {\n            const ctxError = ctx.error;\n\n            if (ctxError !== undefined) {\n                error = ctxError;\n            }\n        }\n\n        return error;\n    },\n\n    filterNotUndefined(call) {\n        for (var key in call) {\n            if (call[key] === undefined) {\n                delete call[key];\n            }\n        }\n\n        if (call.calls !== undefined) {\n            for (var i = 0; i < call.calls.length; i++) {\n                call.calls[i] = this.filterNotUndefined(call.calls[i]);\n            }\n        }\n\n        return call;\n    },\n\n    // sequence converts the finalized calls from a call tree to a call sequence\n    sequence(call, callSequence, availableValueBigInt, traceAddress) {\n        const subcalls = call.calls;\n        delete call.calls;\n\n        call.traceAddress = traceAddress;\n\n        if (call.type === 'call' && call.callType === 'delegatecall') {\n            call.valueBigInt = availableValueBigInt;\n        }\n\n        var newCallSequence = callSequence.concat([call]);\n\n        if (subcalls !== undefined) {\n            for (var i = 0; i < subcalls.length; i++) {\n                const nestedSequenced = this.sequence(\n                    subcalls[i],\n                    newCallSequence,\n                    call.valueBigInt,\n                    traceAddress.concat([i])\n                );\n                newCallSequence = nestedSequenced.callSequence;\n            }\n        }\n\n        return {\n            callSequence: newCallSequence\n        };\n    },\n\n    encodeCallSequence(calls) {\n        for (var i = 0; i < calls.length; i++) {\n            this.encodeCall(calls[i]);\n        }\n\n        return calls;\n    },\n\n    encodeCall(call) {\n        this.putValue(call);\n        this.putGas(call);\n        this.putGasUsed(call);\n\n        return call;\n    },\n\n    putValue(call) {\n        const valueBigInt = call.valueBigInt;\n        delete call.valueBigInt;\n\n        call.value = '0x' + valueBigInt.toString(16);\n    },\n\n    putGas(call) {\n        const gasBigInt = call.gasBigInt;\n        delete call.gasBigInt;\n\n        if (gasBigInt === undefined) {\n            gasBigInt = bigInt.zero;\n        }\n\n        call.gas = '0x' + gasBigInt.toString(16);\n    },\n\n    putGasUsed(call) {\n        const gasUsedBigInt = call.gasUsedBigInt;\n        delete call.gasUsedBigInt;\n\n        if (gasUsedBigInt === undefined) {\n            gasUsedBigInt = bigInt.zero;\n        }\n\n        call.gasUsed = '0x' + gasUsedBigInt.toString(16);\n    }\n}\n"
```

Comment:
Idk what changed here, but is it intentional?

Reply:
🤔 not intentional. Looks like linting changed